dll/win32/shlwapi/autocomp.c

```diff
@@ -0,0 +1,665 @@
+/*
+ * PROJECT:     ReactOS shlwapi
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Implement SHAutoComplete
+ * COPYRIGHT:   Copyright 2020 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#define COBJMACROS
+#include "config.h"
+#include "windef.h"
+#include "winreg.h"
+#include "shldisp.h"
+#include "shlguid.h"
+#define NO_SHLWAPI_STREAM
+#include "shlwapi.h"
+#include "strsafe.h"
+#include "assert.h"
+#include "wine/debug.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(shell);
+
+#define INITIAL_CAPACITY    256
+#define NUM_GROW            64
+#define MAX_TYPED_URLS      50
+
+typedef struct AutoComplete_EnumStringVtbl
```

Comment:
This struct seems exactly identical to `IEnumStringVtbl`, so there doesn't seem to be a need to redefine it.